features/integration/applications.feature

```diff
@@ -21,8 +21,9 @@ Feature: Applications
       And I sign and submit the transaction, saving the txid. If there is an error it is "".
       And I wait for the transaction to be confirmed.
       And I remember the new application ID.
+      Then I get the application info for the current application, and its account matches the app id's hash
```

Comment:
New cucumber test to check that can read an application account from the SDK's `application-info` method.